doc/contributing.md

```diff
@@ -75,6 +75,13 @@ Typical ways to contribute are:
 
 When you open a pull request, a preview of the documentation output is built automatically.
 
+### Prerequisites
+
+* Python 3.8+
+* Python venv module
+* Aspell (for spellchecking)
+* Markdown lint tool (mdl)
```

Comment:
You should use `-` here for the list so it matches the style of the others in this file.

Rather than add things like `mdl` and `venv` to the wordlist, it's best to just put them in a code block.

Given those requirements are specific to the documentation checking, it's probably best to move them into the `Automatic documentation checks` section.

So something like:
```
GitHub runs automatic checks on the documentation to verify the spelling, the validity of links, correct formatting of the Markdown files, and the use of inclusive language.

You can (and should!) run these tests locally as well with the following commands:
- Check the spelling: `make doc-spellcheck`
- Check the validity of links: `make doc-linkcheck`
- Check the Markdown formatting: `make doc-lint`
- Check for inclusive language: `make doc-woke`

To run the above, you will need the following:
- Python 3.8 or higher
- The `venv` python package
- The `aspell` tool for spellchecking
- The `mdl` markdown lint tool
```

Reply:
done, updated doc with comments above.